src/sales/checkout-options.md

```diff
@@ -11,17 +11,45 @@ _[Checkout Options]({% link configuration/sales/checkout.md %})_{:.ee-only}
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the left panel, expand **Sales** and choose **Checkout**.
+1. On the left panel, expand **Sales**.
```

Comment:
This was originally correct. The use must choose that item under sales to display these option.

```suggestion
1. On the left panel, expand **Sales** and choose **Checkout**.
```